proto/agglayer/storage/v0/disabled_network.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package agglayer.storage.v0;
+
+import "google/protobuf/timestamp.proto";
+
```

Comment:
Once disabled always disabled? Do we need a logic/state log to disable/enable network multiple times?